src/main/java/org/opensearch/timeseries/indices/IndexManagement.java

```diff
@@ -477,7 +482,11 @@ public static String getJobMappings() throws IOException {
      */
     public void initJobIndex(ActionListener<CreateIndexResponse> actionListener) {
         try {
-            CreateIndexRequest request = new CreateIndexRequest(CommonName.JOB_INDEX).mapping(getJobMappings(), XContentType.JSON);
+            // AD indices need RAW (e.g., we want users to be able to consume AD results as soon as possible and send out an alert if
+            // anomalies found).
+            Settings replicationSettings = Settings.builder().put(SETTING_REPLICATION_TYPE, DOCUMENT.name()).build();
```

Comment:
This looks good and is a positive change. Would this cause existing detectors to have any issues?

Reply:
It should not. Segment replication should not be enabled for existing indices.